lib/sqlalchemy/dialects/mssql/base.py

```diff
@@ -2590,12 +2591,19 @@ def get_columns(self, connection, tablename, dbname, owner, schema, **kw):
                 columns.c.table_name == tablename,
                 columns.c.table_schema == owner,
             )
+            table_fullname = "%s.%s" % (owner, tablename)
         else:
             whereclause = columns.c.table_name == tablename
+            table_fullname = tablename
         s = sql.select(
             [columns], whereclause, order_by=[columns.c.ordinal_position]
         )
 
+        cc = ischema.computed_columns
+        computedSql = sql.select([cc]).where(
```

Comment:
camelcase here.    if you install the pre-commit hooks, I would *assume* that flake8 would reject these....but maybe im wrong